docs/api/LightroomPartnerAPIsSpec.json

```diff
@@ -1908,7 +1908,7 @@
           "assets"
         ],
         "summary": "Retrieve assets",
-        "description": "Retrieve a list of existing assets that caller owns",
+        "description": "Retrieve a list of existing assets that caller owns. Successful response may also return invalid flag in the rendition link for the asset. A rendition becomes invalid when an asset is edited after rendition has been generated. New renditions (only 2560 and fullsize) can be generated via Generate Renditions API.",
```

Comment:
i think we should also mention: invalid flag in the rendition link only for types 2560 and fullsize